hbt/calibration/default.py

```diff
@@ -174,8 +178,9 @@ def add_calib_cls(name, base, cls_dict=None):
         add_calib_cls("muon_sr_nominal", muon_sr, cls_dict={
             "with_uncertainties": False,
         })
-        # derive met_phi calibrator
-        add_calib_cls("met_phi", met_phi_run2 if self.config_inst.campaign.x.run == 2 else met_phi)
+        if self.config_inst.campaign.x.year!=2024:
```

Comment:
```suggestion
        # TODO: remove if statement wenn met phi corrections are made available
        if self.config_inst.campaign.x.year != 2024:
```

add TODO to not forget to change it as soon as available